examples/sample_run_local_tools_groq.py

```diff
@@ -30,7 +30,7 @@
 
 
 @th.register_local_tool("hello")
-def hello(city: str):
+def hello_tool(city: str):
     """Return Local Time"""
```

Comment:
Change the comment of the function or remove it so it’s not misleading for the user. The comment says the tool returns a local time, but the function does not

Reply:
FIXED